src/data-workspace/data-entry-cell/inputs.js

```diff
@@ -0,0 +1,309 @@
+// Currently it's working okay
+// boolean: accepts 1, 0, 'true', 'false'
+// Ex: if (dv.valueType === boolean) { formValue = dv.value ... etc }
+// does `isEqual` prop help make 1/true and 0/false/'' equal?
+export const BooleanRadios = ({
+    name,
+    syncData,
+    lastSyncedValue,
+    onKeyDown,
+}) => {
+    const yesField = useField(name, {
+        type: 'radio',
+        value: 'true',
+        subscription: { value: true },
+    })
+    const noField = useField(name, {
+        type: 'radio',
+        value: 'false',
+        subscription: { value: true },
+    })
+    // Used for the 'clear' button, but works
+    const clearField = useField(name, {
+        type: 'radio',
+        value: '',
+        subscription: { value: true },
+    })
+    const form = useForm()
+    const fieldState = form.getFieldState(name)
+
+    const clearButtonProps = convertCallbackSignatures(clearField.input)
+    delete clearButtonProps.type
+
+    const handleBlur = () => {
+        const { dirty, valid } = fieldState
+        const value = fieldState.value || ''
+        // If this value has changed, sync it to server if valid
+        if (dirty && valid && value !== lastSyncedValue) {
+            syncData(value)
+        }
+    }
+
+    return (
+        <div
+            className={styles.radioFlexWrapper}
+            // Keydown for keyboard nav works nicely on wrapper
+            onKeyDown={onKeyDown}
+        >
+            <Radio
+                dense
+                label={i18n.t('Yes')}
+                value={'true'}
+                {...convertCallbackSignatures(yesField.input)}
+                onBlur={(_, e) => {
+                    handleBlur()
+                    yesField.input.onBlur(e)
+                }}
+            />
+            <Radio
+                dense
+                label={i18n.t('No')}
+                value={'false'}
+                {...convertCallbackSignatures(noField.input)}
+                onBlur={(_, e) => {
+                    handleBlur()
+                    noField.input.onBlur(e)
+                }}
+            />
+            {fieldState?.value && (
+                <Button
+                    small
+                    secondary
+                    className={styles.whiteButton}
+                    // callback signatures are transformed above
+                    {...clearButtonProps}
+                    // on click, set field value to '', sync, and blur
+                    onClick={() => {
+                        clearField.input.onChange('')
+                        syncData('')
+                        clearField.input.onBlur()
+                    }}
+                    // probably not used, but included as back-up to onClick:
+                    onBlur={() => {
+                        handleBlur()
+                        clearField.input.onBlur()
+                    }}
+                >
+                    {i18n.t('Clear')}
+                </Button>
+            )}
```

Comment:
This 'clear' button is similar to a radio in that it sets this field's value to `''` when clicked